source/assets/javascripts/locastyle/_table.js

```diff
@@ -0,0 +1,104 @@
+var locastyle = locastyle || {};
+
+locastyle.table = (function() {
+  'use strict';
+
+  var config = {
+    selector: {
+      module: '[data-ls-module=table]',
```

Comment:
Two things:
1. When you ommit the double quotations on attribute selectors, this don't work in some browsers. Try put always double quotations to work in everywhere.
2. Another thing: why this module have the name **table**? This module should have a more appropriate name, related to your action and not to the element where it will be used.